pkg/gke/components/GKENodePool.vue

```diff
@@ -464,7 +464,7 @@ export default defineComponent({
           :mode="mode"
           :value="preemptible"
           :disabled="!isNew"
-          @input="$emit('update:preemptible', $event)"
+          @update:value="$emit('update:preemptible', $event)"
```

Comment:
Somehow, this component didn't get this vue3 update to input event syntax. I searched the codebase and didn't find any other components with the same problem.